tests/backend.js

```diff
@@ -35,65 +35,62 @@ test('test error in home route', (t)=>{
   })
 })
 
+test('test returns the file', (t)=>{
+  shot.inject(router, {method:'get', url:'/'}, (res)=>{
+    t.equal(res.statusCode, 200, 'Should respond with a status code of 200');
+    t.end();
+  })
+});
 
-// test('test returns the file', (t)=>{
-//   shot.inject(router, {method:'get', url:'/'}, (res)=>{
-//     const file = ('../public/index.html');
-//     t.equal(res.statusCode, 200, 'Should respond with a status code of 200');
-//     t.equal(res.payload, file, 'should return text/html');
-//     t.end();
-//   })
-// })
 
+const inputTests = [
+  {
+    name: 'valid payload',
+    payload:{
+      language: "javascript",
+      search: "callbacks",
+    },
+    newStatusCode: 200
+  },
+  {
+    name: 'Empty array',
```

Comment:
Can you specify what it should do if it gets an empty array (eg 'should return error' or 'should return some results') ?